src/fastify-plugins/maps.js

```diff
@@ -78,13 +82,17 @@ export async function plugin(fastify, opts) {
 
     fastify.register(SMPServerPlugin, {
       prefix: CUSTOM_MAP_PREFIX,
-      filepath: customMapPath,
+      reader: customMapReader,
     })
   }
 
+  const fallbackMapReader = new ReaderWatch(opts.fallbackMapPath)
+
+  fastify.addHook('onClose', () => fallbackMapReader.close().catch(noop))
+
   fastify.register(SMPServerPlugin, {
     prefix: FALLBACK_MAP_PREFIX,
-    filepath: opts.fallbackMapPath,
+    reader: fallbackMapReader,
```

Comment:
CoMapeo Mobile doesn't really need this to be set up the same way, but figured it would make more sense to do this anyways for the sake of having the flexibility.

Reply:
I like this because I don't want Core to have to rely on Mobile doing things a specific way.